test/hotspot/gtest/utilities/test_align.cpp

```diff
@@ -205,6 +205,7 @@ TEST(Align, alignments) {
   test_alignments<int8_t, int8_t>();
 }
 
+#ifdef ASSERT
```

Comment:
I forgot about the need for this. Glad you caught it.

But include test_fail_alignment under the conditional, to avoid possibility of "defined but unused" warnings.